tasks/setup_RedHat.yml

```
@@ -17,21 +17,17 @@
       ternary(ansible_distribution + 'Linux', ansible_distribution) }}
 
 - name: Add the Okta ASA repo
-  ansible.builtin.copy:
-    # yamllint complains about the length of a few of the lines below,
+  ansible.builtin.yum_repository:
+    # yamllint complains about the length of a couple of the lines below,
     # but there is no way to shorten them.
     #
     # yamllint disable rule:line-length
-    content: |
-      [oktapam-stable]
-      name=Okta PAM Stable - {{ ansible_distribution_tweaked }} {{ ansible_distribution_major_version }}
-      baseurl=https://dist.scaleft.com/repos/rpm/stable/{{ ansible_distribution_tweaked | lower }}/{{ ansible_distribution_major_version }}/$basearch
-      gpgcheck=1
-      repo_gpgcheck=1
-      enabled=1
-      gpgkey=https://dist.scaleft.com/GPG-KEY-OktaPAM-2023
+    baseurl: https://dist.scaleft.com/repos/rpm/stable/{{ ansible_distribution_tweaked | lower }}/{{ ansible_distribution_major_version }}/$basearch
+    description: Okta PAM Stable - {{ ansible_distribution_tweaked }} {{ ansible_distribution_major_version }}
```

...if you make these changes:
```suggestion
    baseurl: >-
      https://dist.scaleft.com/repos/rpm/stable/{{
      ansible_distribution_tweaked | lower }}/{{
      ansible_distribution_major_version }}/$basearch
    description: >-
      Okta PAM Stable - {{ ansible_distribution_tweaked }} {{
      ansible_distribution_major_version }}
```

---

Thoughts on changing the description to:

```yaml
Okta PAM Stable - {{ ansible_distribution_tweaked }}
{{ ansible_distribution_major_version }}
```

Since that still maintains the desired format without having to chop into the substitution.

---

Yes, that's fine.